Resources/Locale/en-US/accent/french.ftl

```diff
@@ -0,0 +1,125 @@
+accent-french-words-1 = yes
+accent-french-words-1-2 = yeah
+accent-french-words-replace-1 = oui
+accent-french-words-2 = no
+accent-french-words-replace-2 = non
+accent-french-words-3 = my
+accent-french-words-replace-3 = mon
+accent-french-words-4 = shit
+accent-french-words-replace-4 = putain
+accent-french-words-5 = passenger
+accent-french-words-replace-5 = passager
+accent-french-words-6 = passengers
+accent-french-words-replace-6 = passagers
+accent-french-words-7 = do
+accent-french-words-replace-7 = faire
+accent-french-words-8 = badly
+accent-french-words-replace-8 = mal
+accent-french-words-9 = man
+accent-french-words-9-2 = guy
+accent-french-words-replace-9 = homme
+accent-french-words-10 = men
+accent-french-words-10-2 = guys
+accent-french-words-replace-10 = hommes
+accent-french-words-11 = woman
+accent-french-words-replace-11 = femme
+accent-french-words-12 = i`m
+accent-french-words-replace-12 = je
+accent-french-words-13 = oh my god
+accent-french-words-replace-13 = oh mon dieu
+accent-french-words-14 = my
+accent-french-words-replace-14 = mon
+accent-french-words-15 = here
+accent-french-words-replace-15 = ici
+accent-french-words-16 = idiot
+accent-french-words-replace-16 = idiote
+accent-french-words-17 = idiots
+accent-french-words-replace-17 = idiots
+accent-french-words-18 = do
+accent-french-words-replace-18 = faire
+accent-french-words-19 = viper
+accent-french-words-replace-19 = vipère
+accent-french-words-20 = officer
+accent-french-words-replace-20 = officier
+accent-french-words-21 = attention
+accent-french-words-replace-21 = l'attention
```

Comment:
l'attention already has an article, annd attention is already a french word so also makes no sense to replacement